dlink-core/src/main/java/com/dlink/explainer/Explainer.java

```diff
@@ -296,7 +296,7 @@ public JobPlanInfo getJobPlanInfo(String statement) {
             }
             return executor.getJobPlanInfoFromDataStream(datastreamPlans);
         } else {
-            return new JobPlanInfo("");
+            throw new RuntimeException("create jobPlan failed, the job not contain insert statement. ");
         }
```

Comment:
Creating job plan fails because this job doesn't contain an insert statement.

Reply:
ok